libraries/chain/include/scorum/chain/services/budgets.hpp

```diff
@@ -96,24 +93,12 @@ template <class InterfaceType, class ObjectType> class dbs_owned_base_budget : p
         FC_CAPTURE_AND_RETHROW((lower_bound_owner_name)(limit))
     }
 
-    typename InterfaceType::budgets_type get_budgets(const account_name_type& owner) const override
+    budgets_type get_budgets(const account_name_type& owner) const override
     {
         try
         {
-            typename InterfaceType::budgets_type ret;
-
-            auto it_pair = this->db_impl()
-                               .template get_index<budget_index<ObjectType>>()
-                               .indices()
-                               .template get<by_owner_name>()
-                               .equal_range(owner);
-
-            for (auto it = it_pair.first, it_end = it_pair.second; it != it_end; ++it)
-            {
-                ret.push_back(std::cref(*it));
-            }
-
-            return ret;
+            return this->template get_range_by<by_owner_name>(::boost::lambda::_1 >= owner,
+                                                              ::boost::lambda::_1 <= owner);
```

Comment:
Not readable sequance. Use (owner <=::boost::lambda::_1,  ::boost::lambda::_1 <= owner)